src/Microsoft.Data/Microsoft.Data.DataFrame.csproj

```diff
@@ -99,6 +99,11 @@
       <AutoGen>True</AutoGen>
       <DependentUpon>BaseColumn.Computations.tt</DependentUpon>
     </Compile>
+    <Compile Update="ColumnArithmeticTemplate.cs">
```

Comment:
I don't think this file exists.  Typically `.ttinclude` files don't generate their own `.cs` file. (At least in my experience.)
This change can be reverted.